packages/postgres-database/src/simcore_postgres_database/models/_common.py

```diff
@@ -0,0 +1,22 @@
+import sqlalchemy as sa
+
+
+def column_created() -> sa.Column:
+    return sa.Column(
+        "created",
+        sa.DateTime(),
+        nullable=False,
+        server_default=sa.sql.func.now(),
+        doc="Timestamp auto-generated upon creation",
+    )
+
+
+def column_modified():
```

Comment:
Minor: This is mostly me.

Regarding below names, I would also include the type, something like `column_created_datetime`.
I find it more easy to read (avoids clicking and inspect new module).

```python
def column_created() -> sa.Column:
...
def column_modified():
```